app/javascript/src/assets/scss/widgets/_tabs.scss

```diff
@@ -1,50 +1,52 @@
 .tabs {
-  border-width: 0;
   @include padding($zero $space-normal);
   @include border-normal-bottom;
+  border-width: 0;
```

Comment:
@kirillarutyunov This broke the main page style. Please see the screenshot attached. Could you please take a look at it. 

<img width="424" alt="Screenshot 2019-10-05 16 36 41" src="https://user-images.githubusercontent.com/2246121/66252450-5d684e80-e78e-11e9-8df0-6e239c873098.png">